src/System.AppContext/src/System/AppContext.cs

```diff
@@ -44,6 +45,19 @@ public static object GetData(string name)
             return null;
         }
 
+        public static event UnhandledExceptionEventHandler UnhandledException
```

Comment:
We will want to move the AppContext implementation to System.Private.CoreLib for CoreRT as well.